filters/adjustment/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2013-2017 Mathew Groves, Chad Engler
```

Comment:
Change copyright?

Reply:
After all of my filters be merged , I will create a new PR for change all of them.

Reply:
Okay 👍